manifests/a/AgileBits/1Password/CLI/2.30.0/AgileBits.1Password.CLI.installer.yaml

```
@@ -0,0 +1,26 @@
+# Created with komac v2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: AgileBits.1Password.CLI
+PackageVersion: 2.30.0
```

```suggestion
PackageVersion: 2.30.0
InstallerLocale: en-US
```

It seems `InstallerLocale` is different between contributors in different countries or regions. _Komac_ on my device will inspect it as `zh-CN`.